react/features/invite/components/add-people-dialog/native/AddPeopleDialog.js

```diff
@@ -318,13 +318,11 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
                 }
             });
 
-            const items = this.state.inviteItems.concat(selectableItems);
-
-            // $FlowExpectedError
-            selectableItems = _.orderBy(items, [ 'name' ], [ 'asc' ]);
+            selectableItems = _.sortBy(selectableItems, [ 'name', 'number' ]);
+            const sortedInviteItems = _.sortBy(this.state.inviteItems, [ 'name', 'number' ]);
```

Comment:
Do you need to sort these again?

Reply:
Yes. After the first run it becomes ordered, after the second run, it gets ordereded